css-view-transitions-1/Overview.bs

```diff
@@ -1101,19 +1070,81 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 				except the [=view-transition pseudo-elements=],
 				are not painted (as if they had visibility: hidden)
 				and do not respond to hit-testing (as if they had pointer-events: none) until |new| exists.
+
+			1. If neither of |capturedElement|'s [=captured element/old image=] or [=captured element/new element=] is null:
```

Comment:
I prefer "If both ... are not null", since it sounds more in the positive? Just a nit though.